@here/harp-mapview/lib/Tile.ts

```diff
@@ -1231,7 +1231,8 @@ export class Tile implements CachedResource {
                         {
                             technique,
                             level: Math.floor(displayZoomLevel),
-                            fog: this.mapView.scene.fog !== null
+                            fog: this.mapView.scene.fog !== null,
+                            skipExtraProps: ["map", "normalMap", "displacementMap"]
```

Comment:
Shouldn't `(map|normalMap|...)Properties` be exluded too ?

Reply:
True, good catch